templates/index.html.twig

```diff
@@ -56,12 +46,12 @@
                     <div class="fr-card">
                         <div class="fr-card__body">
                             <div class="fr-card__content">
-                                <h3 class="fr-card__title">{{ 'landing.how-it-works.gps.title'|trans }}</h3>
+                                <h3 class="fr-card__title">{{ 'landing.for-gps.title'|trans }}</h3>
```

Comment:
Cette clé de traduction n'a pas été trouvée

Dans la toolbar symfony j'ai ceci :

<h2>Messages</h2>

            
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                        
        <div class="sf-tabs" data-processed="true"></div>
            

            

            <div class="block" id="tab-0-2" role="tabpanel" aria-labelledby="tab-0-2">
                

                <div class="tab-content">
                    <p class="help">
                        These messages are not available for the given locale and cannot
                        be found in the fallback locales. Add them to the translation
                        catalogue to avoid Symfony outputting untranslated contents.
                    </p>

                                                                            

Locale | Domain | Times used | Message ID | Message Preview
-- | -- | -- | -- | --
fr | messages | 1 | landing.for-gps.title | landing.for-gps.title

</div></div>

Reply:
j'ai corrigé merci !